### ISSUE TITLE
Several optional price-sources and additional functions

### ISSUE THREAD

**Issue by MarkoPaasila** [Codaone/DEXBot#271]
This is a checklist for a set of features, each of which can be implemented separately.

I hope we can implement the following price-sources to be used by any strategy. In the Relative Orders configuration dialog, the user should be able to select one source:
[x] Center of closest opposite orders
[ ] External price feed (exchanges, coinmarketcap, etc. This might need a separate modal)
[ ] Last own trade
[ ] Last market trade
[ ] Center of opposite market orders from a certain depth (relative to own order sizes?)
[ ] MPA price feed from BitShares

In addition, the following options should be available to enhance the price data:
[ ] Time based moving average (configurable seconds)
[ ] Trades based moving average (configurable amount of trades)
[ ] Time based weighted moving average (configurable weight)
[ ] Volume based weighted moving average (configurable weight)

In total this is pretty big, but I want there to be a big picture.

**Comment by bitfag** [comment]
* [x] External price feed

**Comment by thehapax** [comment]
@MarkoPaasila I am moving this back to Status:Available as we are now back in development. let me know if you have any objections.

**Comment by thehapax** [comment]
@PermieBTS this task should be broken into smaller ones.

**Comment by bitfag** [comment]
May be an Epic issue according to scrum terms.

**Comment by MarkoPaasila** [comment]
Yes Epics are better since we use Scrum now. Can you close this issue once the epic is written?

**Comment by thehapax** [comment]
@MarkoPaasila @bitfag I am assigning this epic issue to @PermieBTS

**Comment by bitfag** [comment]
@thehapax maybe add a new label "Epic" to be able to quickly find these issues?

**Comment by PermieBTS** [comment]
https://www.yodiz.com/blog/what-is-epic-in-agile-methodology-definition-and-template-of-epic/

**Comment by MarkoPaasila** [comment]
Last trade as new cp: #611
